cmd/opampsupervisor/supervisor/commander/commander.go

```diff
@@ -52,6 +55,13 @@ func (c *Commander) Start(ctx context.Context) error {
 		return nil
 	}
 
+	if len(c.doneCh) > 0 {
+		<-c.doneCh
+	}
```

Comment:
is draining the done channel necessary here? It seems like this channel is remade later in this function.

Reply:
Thanks for pointing out. It was supposed to be removed.